articles/active-directory-b2c/service-limits.md

```diff
@@ -32,7 +32,7 @@ The number of users able to authenticate through an Azure AD B2C tenant is gated
 
 |Category |Limit    |
 |---------|---------|
-|Maximum requests per IP per Azure AD B2C tenant       |6,000/5min          |
```

Comment:
Even though rate limiting layer has some common pieces for all types of tenants - Entra ID, Entra External ID and Azure AD B2C, the rate limits themselves are different. So, this line item needs to be B2C specific.